README.md

```diff
@@ -35,6 +35,10 @@ args:
 dirs:
   - var/data/tmp
   - var/log
+# OPTIONAL - A map of configurations of secondary processes to launch
+secondaries:
+  name:
+    StaticLauncherConfig
```

Comment:
Maybe one copy & pastable example, possibly even mentioning envoy (to help people understand the motivation)